zebra/zebra_ptm.c

```diff
@@ -1295,7 +1295,6 @@ static void zebra_ptm_send_bfdd(struct stream *msg)
 	}
 
 	stream_free(msgc);
-	stream_free(msg);
```

Comment:
weird - I'm not seeing where the caller frees this: is it hidden in a macro somewhere?

Reply:
by "caller" I meant higher up in the call chain; it's freed in `zserv_process_messages`

Reply:
doh - yes, quite - thanks